app-backend/api/src/main/scala/uploads/Credentials.scala

```diff
@@ -51,7 +51,7 @@ object CredentialsService extends Config with LazyLogging {
     stsRequest.setRoleArn(scopedUploadRoleArn)
     stsRequest.setRoleSessionName(s"upload${uploadId}")
     stsRequest.setWebIdentityToken(jwt)
-    stsRequest.setDurationSeconds(900)
+    stsRequest.setDurationSeconds(3600)
```

Comment:
Could you add a comment here about this being the max value?

Reply:
@alkamin sure.